apps/aecore/lib/aecore/peers/sync.ex

```diff
@@ -83,41 +83,37 @@ defmodule Aecore.Peers.Sync do
 
   @spec add_peer_blocks_to_sync_state(String.t(), binary()) :: :ok
   def add_peer_blocks_to_sync_state(peer_uri, from_block_hash) do
-    if Chain.has_block?(from_block_hash) do
-      remove_running_task(peer_uri)
+    with false <- Chain.has_block?(from_block_hash),
```

Comment:
The if was cleaner I think, but this can stay as is.

Reply:
Well, having in mind that the `Sync` will be rewritten so this is not important anymore.